### ISSUE TITLE
Paragraph spaces are not correctly rendered in Word arrays

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#5]
```
Paragraph spaces are not correctly rendered in Word arrays.
```

Original issue reported on code.google.com by `jerome.m...@gmail.com` on 9 Mar 2011 at 1:01

**Comment by GoogleCodeExporter** [comment]
Original comment by `angelo.z...@gmail.com` on 14 Apr 2011 at 12:51
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the 1.0.0.
```

Original comment by `angelo.z...@gmail.com` on 2 Nov 2012 at 3:13
- Changed state: **Fixed**
